### ISSUE TITLE
Stuck at "Syncing Mint and Robinhood... This window will automatically close when the sync is complete"

### ISSUE THREAD

**Issue by hannahvu** [pkmnct/robinhood-mint-sync-chrome#17]
I've been stuck on this page on Robinhood for over 10 minutes. I logged into just Mint initially and was told that the sync was starting. However, I was stuck on that page for a long time so I logged into both Mint and Robinhood to try to debug and I'm still stuck on that page. Please help

**Comment by ryanwnzl** [comment]
Same issue today for me

**Comment by TimGodfrey** [comment]
getting the following for me: 

Robinhood Mint Sync for Chrome
Syncing Mint with Robinhood.

×

Robinhood Mint Sync for Chrome
Sync Complete! Reload to see the change.


Robinhood Mint Sync for Chrome
Sync performed in the last hour. Not syncing.

**Comment by pkmnct** [comment]
Try signing into Robinhood.com manually before initiating the sync. I will look into why the login page is not being detected

**Comment by pkmnct** [comment]
@TimGodfrey @ryanwnzl @hannahvu Just to confirm, did signing in to Robinhood.com manually before starting the sync work? I have a fix for the login page detection in an upcoming update (#20), but I wanted to confirm that was your issue.

**Comment by pluong94** [comment]
Hi, 

I am still having this issue. I am logged into robinhood and mint. From mint overview, using 3.1.2 version of the app, I see nothing has been edittted in my mint account.

I can confirm there is not "property" called robinhood. 

What can i do here? Thanks in advance.

**Comment by superpowered** [comment]
@pluong94 Do any of the extension prompts appear in the top right of the screen? (there should either be a prompt asking you to set up, or a prompt about it attempting to sync)

If so: Does it try actually syncing anything? (E.g. Do any other tabs open in chrome? When syncing, one tab should open for robinhood, and then another tab should open for Mint Settings)

If so: Can you turn on "Enable Debug" in the extension settings and then post what you see in the inspector console on the new tabs?

**Comment by pluong94** [comment]
Hey Andrew,

Outside of the initial extension prompt asking me to set up, no additional
prompts appear. Once I click the set up from the extension prompt, it
brings me to this black screen as others have seen.

Nothing actually tries syncing.

I’ll do the debug when I get to my computer!

**Comment by pluong94** [comment]
@superpowered 

What am i suppose to be posting? I have turned on "enable debug" but am not sure what I am looking for from inspecting the background page.

**Comment by pluong94** [comment]
@superpowered  

Actually, I think i found it?

Robinhood Mint Sync - [Content](Mint - Properties - Check) initialized
debug.ts:33 Robinhood Mint Sync - [Content](Mint - Triangle Fix) initialized
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Using default sandbox factory...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 All core plugins started...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Plugins registered...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Layout activated
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Layout activated
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Application before-ready completed...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Application ready...
app-284d2e17557964650d51748b0a1ee211.js:11016 LD: [error] Ignoring unknown config option "baseTimeout"
r @ app-284d2e17557964650d51748b0a1ee211.js:11016
inpage.js:1 MetaMask: MetaMask will soon stop reloading pages on network change.
For more information, see: https://docs.metamask.io/guide/ethereum-provider.html#ethereum-autorefreshonnetworkchange 
Set 'ethereum.autoRefreshOnNetworkChange' to 'false' to silence this warning.
(anonymous) @ inpage.js:1
inpage.js:1 MetaMask: MetaMask will soon stop reloading pages on network change.
For more information, see: https://docs.metamask.io/guide/ethereum-provider.html#ethereum-autorefreshonnetworkchange 
Set 'ethereum.autoRefreshOnNetworkChange' to 'false' to silence this warning.
(anonymous) @ inpage.js:1
inpage.js:1 MetaMask: MetaMask will soon stop reloading pages on network change.
For more information, see: https://docs.metamask.io/guide/ethereum-provider.html#ethereum-autorefreshonnetworkchange 
Set 'ethereum.autoRefreshOnNetworkChange' to 'false' to silence this warning.
(anonymous) @ inpage.js:1
app-284d2e17557964650d51748b0a1ee211.js:531 Object
printToConsole @ app-284d2e17557964650d51748b0a1ee211.js:531
send @ app-284d2e17557964650d51748b0a1ee211.js:449
warn @ app-284d2e17557964650d51748b0a1ee211.js:650
MintEventSource.onerror @ app-284d2e17557964650d51748b0a1ee211.js:5022
E @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
S @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
g.onreadystatechange @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
_abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
b.abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
c.cancel @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563 Uncaught Error: No activity within 45000 milliseconds. Reconnecting.
    at U (libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563)
    at libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
app-284d2e17557964650d51748b0a1ee211.js:531 Object
printToConsole @ app-284d2e17557964650d51748b0a1ee211.js:531
send @ app-284d2e17557964650d51748b0a1ee211.js:449
warn @ app-284d2e17557964650d51748b0a1ee211.js:650
MintEventSource.onerror @ app-284d2e17557964650d51748b0a1ee211.js:5022
E @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
S @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
g.onreadystatechange @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
_abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
b.abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
c.cancel @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563 Uncaught Error: No activity within 45000 milliseconds. Reconnecting.
    at U (libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563)
    at libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
app-284d2e17557964650d51748b0a1ee211.js:531 Object
printToConsole @ app-284d2e17557964650d51748b0a1ee211.js:531
send @ app-284d2e17557964650d51748b0a1ee211.js:449
warn @ app-284d2e17557964650d51748b0a1ee211.js:650
MintEventSource.onerror @ app-284d2e17557964650d51748b0a1ee211.js:5022
E @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
S @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
g.onreadystatechange @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
_abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
b.abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
c.cancel @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563 Uncaught Error: No activity within 45000 milliseconds. Reconnecting.
    at U (libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563)
    at libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
app-284d2e17557964650d51748b0a1ee211.js:531 Object
printToConsole @ app-284d2e17557964650d51748b0a1ee211.js:531
send @ app-284d2e17557964650d51748b0a1ee211.js:449
warn @ app-284d2e17557964650d51748b0a1ee211.js:650
MintEventSource.onerror @ app-284d2e17557964650d51748b0a1ee211.js:5022
E @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
S @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
g.onreadystatechange @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
_abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
b.abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
c.cancel @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563 Uncaught Error: No activity within 45000 milliseconds. Reconnecting.
    at U (libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563)
    at libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
app-284d2e17557964650d51748b0a1ee211.js:531 Object
printToConsole @ app-284d2e17557964650d51748b0a1ee211.js:531
send @ app-284d2e17557964650d51748b0a1ee211.js:449
warn @ app-284d2e17557964650d51748b0a1ee211.js:650
MintEventSource.onerror @ app-284d2e17557964650d51748b0a1ee211.js:5022
E @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
S @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
g.onreadystatechange @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
_abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
b.abort @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
c.cancel @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563 Uncaught Error: No activity within 45000 milliseconds. Reconnecting.
    at U (libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563)
    at libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
U @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
(anonymous) @ libs-f0625f303abccdbf22c19e5c5570eeb9.js:13563
DevTools failed to load SourceMap: Could not load content for chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/sourcemaps/disable-console.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/sourcemaps/contentscript.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/sourcemaps/inpage.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/sourcemaps/disable-console.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/sourcemaps/contentscript.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/sourcemaps/inpage.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for https://uxfabric.intuitcdn.net/react-intl/2.9.0/dist/react-intl.min.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/4.b654202ae91c53829b85d18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/14.ce640cf1721968f3c5b3d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/18.efbd929eb473ad8cff61d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/7.e42941735b26bd69ef18d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/9.c65dcdb5c3915ee5ce23d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/8.6f2d99fff9e1b7b0f03fd18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/4.57fa3ed173aafa6b570ed18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/13.4fbb00ab1ca1b1009933d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/10.2ecfa38d105ed6760b13d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/11.7a0f45a7d2c65ef89f9bd18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/12.9ad58fd2577b1de58bded18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/15.64af85c9e122bacc29d1d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/js-widgets-carousel-carouselwidget~41603be3.236bce984bb2d6790ee1d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/js-widgets-marketplace-marketplacewidget~c6795cd9.512f9b4a6bfd250e493ad18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/17.cb7f39b12d23d2c3aa38d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/6.be399d54aff726ae89c1d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/js-widgets-singleoffer-singlewidget~86990be6.76d60022a70267fa22b7d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/7.a657df9b6e228aa710a4d18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/17.7eec833c13b00876470bd18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/9.3527257f88ba21014cc2d18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/11.829f15aa2f678f793e1ed18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/promotions-personalized-offers-ui~493df0b3.bb356f07c9bad886bad5d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://my.nanorep.com/widget/scripts/sourcemaps/floating-widget.js.map;: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/2.7d309b61f7aa40bf0d41d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/5.9e8aeab44f01a8013fc5d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/1.aee520f6685234abe318d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/0.db011e012ad1f652c006d18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/3.934abe30ca5c1952a65dd18563df.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/15.989d8db947937b397286d18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/6.94685b886d02d47737b7d18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://plugin.intuitcdn.net/promotions-personalized-offers-ui/18.9c0fac24cfcba3f08acad18563df.css.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://images.mint.com/web-client/min/js/ids.js.map;: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://images.mint.com/web-client/min/js/ldclient.min.js.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by superpowered** [comment]
Okay, so was able to recreate after making a "Fresh" account with no properties....

https://github.com/pkmnct/robinhood-mint-sync-chrome/pull/27 This *should* address it.

@pluong94 I'm assuming you had no other properties in the account?

**Comment by pluong94** [comment]
Hi,

@superpowered  Thanks, yeah it just started working for me. and No, i had no other properties in the account.

**Comment by pkmnct** [comment]
@pluong94 @superpowered I created a new issue for the account with no properties (#31).
